1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,25 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach(n => {
```

Comment:
your logic here is good but I think you should take more care to indent your code more neatly